nextflow.config

```
@@ -200,25 +179,24 @@ profiles {
     test_full { includeConfig 'conf/test_full.config' }
 }
 
-// Set default registry for Apptainer, Docker, Podman and Singularity independent of -profile
-// Will not be used unless Apptainer / Docker / Podman / Singularity are enabled
-// Set to your registry if you have a mirror of containers
-apptainer.registry   = 'quay.io'
-docker.registry      = 'quay.io'
-podman.registry      = 'quay.io'
-singularity.registry = 'quay.io'
+// Load nf-core custom profiles from different Institutions
+includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/nfcore_custom.config" : "/dev/null"
 
-// Nextflow plugins
-plugins {
-    id 'nf-validation@1.1.3' // Validation of pipeline parameters and creation of an input channel from a sample sheet
-}
+// Load nf-core/pairgenomealign custom profiles from different institutions.
+// includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/pipeline/pairgenomealign.config" : "/dev/null"
```

```suggestion
includeConfig !System.getenv('NXF_OFFLINE') && params.custom_config_base ? "${params.custom_config_base}/pipeline/pairgenomealign.config" : "/dev/null"
```

This file already exists so this doesn't need to be uncommented: https://github.com/nf-core/configs/blob/master/pipeline/pairgenomealign.config